host_applications/linux/apps/raspicam/RaspiStill.c

```diff
@@ -216,7 +215,6 @@ static void store_exif_tag(RASPISTILL_STATE *state, const char *exif_tag);
 #define CommandSignal       16
 #define CommandGL           17
 #define CommandGLCapture    18
-#define CommandSettings     19
```

Comment:
Does it cause a problem having a gap in the numbers? Do we rewrite this as an enum definition?

Reply:
No problem with gaps, but I'll change to enums in a different PR anyway. unless I remember why I did that way originally...